docs/conf/portland/2018/news/volunteers-shirts-tickets.rst

```diff
@@ -0,0 +1,66 @@
+:template: {{year}}/{{templatecode}}/generic.html
+
+.. post:: March 19, 2018
+   :tags: portland-2018, volunteers, shirts, tickets
+
+Call for Volunteers, Shirts on Sale, Tickets Update
+===================================================
+
+Greetings, fellow documentarians! The Portland conference is just short of two months away, and we can't wait to see you all soon! Here are the latest updates from the conference team to help you prepare.
+
+Shirts on sale
+--------------
+
+As you probably noticed, we are not including shirts with your conference ticket this year.
+This is due partially to the logistics of sizing, ordering, and managing stock, but also because of the inevitable leftovers that we find ourselves with every year.
+
+Inspired by our friends at DjangoCon US, we decided to shake things up and launch a separate sales channel for shirts, if you have your heart set out on them.
+You can now visit our `Write the Docs Portland 2018 Pop-Up Shop <https://teespring.com/wtd-portland-2018-shirts>`_ and order this year's branded shirt.
+The campaign will run until **April 15th** to allow for timely delivery of your shirts ahead of the conference.
+
+And don't worry, you'll still have a Write the Docs Portland 2018 souvenier from the conference even if you decide not to order a shirt.
+We'll send another update once we finalized the design and product.
```

Comment:
Do we need to update people on the tote bag? Seems like an overpromise we might not fulfill. I don't think "fancy new swag!" is something we're promoted in the past, and might be weird to promote this year.